mvpa2/algorithms/hyperalignment.py

```diff
@@ -340,14 +352,15 @@ def __call__(self, datasets):
             # We need to construct new mappers which would chain
             # zscore and then final transformation
             if params.alpha < 1:
-                return [ChainMapper([zm, wm, m]) for zm, wm, m in zip(zmappers, wmappers, mappers)]
+                mappers = [ChainMapper([zm, wm, m]) for zm, wm, m in zip(zmappers, wmappers, mappers)]
             else:
-                return [ChainMapper([zm, m]) for zm, m in zip(zmappers, mappers)]
-        else:
-            if params.alpha < 1:
-                return [ChainMapper([wm, m]) for wm, m in zip(wmappers, mappers)]
-            else:
-                return mappers
+                mappers = [ChainMapper([zm, m]) for zm, m in zip(zmappers, mappers)]
+        elif params.alpha < 1:
+                mappers = [ChainMapper([wm, m]) for wm, m in zip(wmappers, mappers)]
```

Comment:
no it is not ;-) ... pushed commit with a "fix" and also has hidden away `_svd_mapper` - IMHO no need to expose it until we need to